lib/generation/generators/symbols/pb_instancesym_gen.dart

```diff
@@ -102,43 +65,119 @@ class PBSymbolInstanceGenerator extends PBGenerator {
       // Try to find master by looking for the master's SYMBOL_ID
       masterSymbol = PBSymbolStorage().getSharedMasterNodeBySymbolID(UUID);
     }
-    // file could have override names that don't exist?  That's really odd, but we have a file that does that.
-    if (masterSymbol == null) {
+
+    return masterSymbol;
+  }
+
+  String genSymbolInstance(
+      String UUID,
+      Map<String, PBSymbolInstanceOverridableValue> mapOverrideValues,
+      Map<String, PBSymbolMasterParameter> mapParameterValues,
+      PBGenerationViewData managerData,
+      { bool topLevel = true,
+        String UUIDPath = ''}) {
+    if ((UUID == null) || (UUID == '')) {
       return '';
     }
 
-    assert(masterSymbol != null,
-        'Could not find master symbol with UUID: ${UUID}');
     var buffer = StringBuffer();
-    buffer.write('${masterSymbol.friendlyName}(constraints, ');
-    for (var ovrValue in overrideValues) {
-      var ovrUUIDStrings = ovrValue.UUID.split('/');
-      if ((ovrUUIDStrings.length == depth + 1) &&
-          (ovrUUIDStrings[depth - 1] == overrideUUID)) {
-        var ovrUUID = ovrUUIDStrings[depth];
-        switch (ovrValue.type) {
+    var masterSymbol = getMasterSymbol(UUID);
+
+    // file could have override names that don't exist?  That's really odd, but we have a file that does that.
+    if (masterSymbol == null) {
+      log.error(' Could not find master symbol for UUID:: $UUID');
+      return 'Container(/** This Symbol was not found **/)});';
+    }
+
+    var symName = masterSymbol.name;
```

Comment:
A suggestion here is to configure line 92 to the following:
`var symName = masterSymbol.name.pascalCase`

This is because in line `98`, when we format the label, we sometimes get incorrect variable names/imports. For instance `SmallMajorCard/loved` should become `SmallMajorCardLoved` as a reference, and `small_major_card_loved` as a file reference. However, the output displays as `SmallMajorCardloved` as reference and `small_major_cardloved` as file reference:

![Screen Shot 2021-05-26 at 11 36 43 AM](https://user-images.githubusercontent.com/42812006/119705999-a747f180-be16-11eb-815b-947a97f334e8.png)

Please let me know what you think of this if you have any thoughts 🤔

Reply:
Sounds good to me.  Do you make changes or do I?

Reply:
I would make the changes myself, but I'm not sure how to push my changes to the PR rather than directly to the dev branch here in `parabeac-core` 🤔 . 

I'm also noticing that specifically for something like `person/4`, regardless of whether we make it pascalCase or not, it exports as `person4` rather than `person_4`, I'm also looking into how to solve this issue.

Reply:
See below, commit automatically added.